pkg/usersignup/usersignup.go

```diff
@@ -1,6 +1,12 @@
 package usersignup
 
 import (
+	"crypto/md5" // nolint:gosec
```

Comment:
This change in `usersignup.go` doesn't seem to be related to the builder.

Reply:
the notification_builder_test uses the changes in usersignup.go

Reply:
OK. Then I guess you don't need to keep this code in the host operator anymore and can switch to common here: https://github.com/codeready-toolchain/host-operator/pull/629/files

Reply:
done